RPG Wizardry/Assets/Scripts/Sorcery/SpellPage.cs

```diff
@@ -11,6 +11,11 @@ public class SpellPage
         /// Whether the Spell on this Page has been Unlocked
         /// </summary>
         public bool Unlocked { get; private set; }
+
+        /// <summary>
+        /// The dust cost for the spell to unlock.  
+        /// </summary>
+        public uint DustCost { get; private set; }
```

Comment:
Should this be in SpellData-ScriptableObject? (I did that in my (open) branch)

Reply:
Added a TODO for this in the code.